pkg/confgen/flowlogs2metrics_config.go

```diff
@@ -56,6 +56,7 @@ func (cg *ConfGen) GenerateFlowlogs2PipelineConfig() *config.ConfigFileStruct {
 	}
 	if len(cg.promMetrics) > 0 {
 		metricsNode.EncodePrometheus("encode_prom", api.PromEncode{
+			Address: cg.config.Encode.Prom.Address,
```

Comment:
Check here that cg.config.Encode.Prom.Address is actually defined (is not nil). If not, use 0.0.0.0.

Reply:
This is not necessary. If address is not specified, then we end up with ":<port>", just like we used to have earlier, and it works OK.